.github/workflows/firesim-cull-instances.yml

```diff
@@ -0,0 +1,25 @@
+name: firesim-cull-instances
```

Comment:
So does this just run globally, or per branch? The existing implementation kinda sucks because it's per branch (well, it'll run on dev once dev is merged to master).

Reply:
This runs on the latest commit from the main branch by default... however, I changed it to checkout the `dev` branch then run the CI code.

Reply:
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#schedule